src/config.py

```diff
@@ -0,0 +1,40 @@
+
+import os
+
+
+class Config:
+    '''
+    Configuration for the application.
+
+    Any values that need to be exposed through Flask MUST be in `UPPERCASE`.
+    '''
+
+    # Flask Configuration
+
+    HOST: str = '0.0.0.0'
+    PORT: int = 2402
+
+    SECRET_KEY: str = 'bf7fe7847aa5f10778de0340d4b7cb5163d2727f95801ba0'
```

Comment:
Is the SECRET_KEY a config setting we want users to be able to change?

Reply:
It's a part of configuration and it's used for encryption by flask; changing it just makes uploading fail so I think if we just don't mention it in the documentation it'll be fine.